project/ScalafixBuild.scala

```diff
@@ -227,28 +224,9 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     publishLocalTransitive := Def.taskDyn {
       val ref = thisProjectRef.value
       publishLocal.all(ScopeFilter(inDependencies(ref)))
-    }.value,
-    crossPublishLocalBinTransitive := {
-      val currentState = state.value
-      val ref = thisProjectRef.value
-      val versions = crossScalaVersions.value
```

Comment:
https://github.com/sbt/sbt-projectmatrix/blob/e46c2e3c3a4eba186a204303834e5d01893455b7/src/main/scala/sbt/internal/ProjectMatrix.scala#L145 has exactly what we need to iterate over versions so we can get rid of this (which would have required some tweaks anyway since `crossScalaVersions` is not set anymore)